docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -5343,3 +5344,92 @@ _Available in Fleet Basic_
 ```
 
 ---
+
+## Translator
+
+### Translate IDs
+
+_Available in Fleet Basic_
```

Comment:
```suggestion
```
Fleet Basic is the paid offering (confusing, we know -- changing this) and this endpoint is available on all tiers.